### PR TITLE
feat: use different cache key for command bar

### PR DESCRIPTION
**Pull request by sjaanus**
Now command bar and search do not share same cache, so they wont conflict anymore.

### CODE REVIEW

File: frontend/src/hooks/api/getters/useFeatureSearch/useFeatureSearch.ts [hunk 1]
```diff
@@ -55,17 +55,19 @@ const createFeatureSearch = () => {
     return (
         params: SearchFeaturesParams,
         options: SWRConfiguration = {},
+        cachePrefix: string = '',
     ): UseFeatureSearchOutput => {
         const { KEY, fetcher } = getFeatureSearchFetcher(params);
```
Comment (daveleek) [inline frontend/src/hooks/api/getters/useFeatureSearch/useFeatureSearch.ts:60]: is the KEY in use here after this change?

Comment (sjaanus) [reply frontend/src/hooks/api/getters/useFeatureSearch/useFeatureSearch.ts:60]: Yes, on line 61.

Comment (daveleek) [reply frontend/src/hooks/api/getters/useFeatureSearch/useFeatureSearch.ts:60]: Sigh, blink and you miss it 😅

**Review by daveleek — APPROVED** [review]
LG!

**Pull request merged**